code/modules/surgery/organs/eyes.dm

```diff
@@ -5,7 +5,7 @@
 	organ_tag = "eyes"
 	parent_organ = "head"
 	slot = "eyes"
-	var/eye_colour = "#000000"
+	var/eye_colour = "#000000" // Should not be null
```

Comment:
Might be a better idea to check if the eye_colour is null on init/new and then set it to this instead, future-proof!

Reply:
Added the lazy programmer protection. Not sure how I feel about it but I know it will happen again so why not add it